english/yoga/benefits/yoga_benefits.md

```diff
@@ -12,4 +12,4 @@ Here we provide a list of benefits:
 2. Increased muscle Strength and tone
 3. Enhanced gluten sensitivity 
 4. Kombucha cravings
-5. 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.